.travis.yml

```diff
@@ -1,9 +1,23 @@
-dist: xenial  # required for Python >= 3.7
 language: python
 
-python:
-  - "2.7"
-  - "3.7"
+matrix:
+  include:
+    - name: "Python 2.7 on Xenial Linux"
+      os: linux
+      dist: xenial
+      python: 2.7
+    - name: "Python 3.7 on Xenial Linux"
+      os: linux
+      dist: xenial
+      python: 3.7
+    - name: "Python 3.7.4 on Windows"
+      os: windows
+      language: node_js
```

Comment:
Why do we have to set language to node_js and not python?

Reply:
Because travis doesn't support set python on windows. But we need npm, so I set the language to node_js
https://docs.travis-ci.com/user/languages/python/

Reply:
I see, makes sense. Thanks for adding!